docs/tasks/semester-checklist.rst

```diff
@@ -1,9 +1,39 @@
 Semester Checklist
 ==================
-
 A quick checklist of tasks that need to be completed at the start of every semester.
 
--  Reserve the meeting room by speaking to the lab manager (requires a signature from the president and vice president). Get set up with swipe access if need be.
+Before Semester
+---------------
+Tasks to do during the break.
+
+-  Hold weekly/bi-weekly eboard meetings.
+-  Discuss plans and brainstorm for next semester.
+-  Reserve meeting space.
+-  Triage and review tasks from previous semester to plan for coming semester (see project board).
+
+Before Fall Semester
+----------------------
+Tasks which only need to be completed before the fall semester.
+
+-  Get all eboard members financially certified through Campus Life.
+-  Plan for the Activities Gala.
 -  Complete any outstanding eboard :doc:`transition tasks <../administration/eboard-onboarding-offboarding>`
--  Set up a meeting with the club faculty adviser to discuss plans for the semester
--  Update the website and thelink for the new semester
+
+At Start of Semester
+--------------------
+Tasks for last few weeks of break.
+
+-  Set up a meeting with the club faculty adviser to discuss plans for the semester.
+-  Create a poster for the start of the semester.
+-  Present at Activities Gala (fall semester only).
+-  Send out a welcome email to people from the gala and returning members.
+-  Find a weekly eboard meeting time based on schedules of all eboard members in Rochester.
+
+First Week of Semester
+----------------------
+Tasks which should be done the first few weeks of classes.
+
+-  Have an eboard member attend the All Clubs Meeting.
+-  Get ready for the first few meetings.
+-  Send first meeting emails.
+-  Request budget from campus life.
```

Comment:
This task is usually done towards the end of the spring semester (unless I missed a recent policy change). For example, I filled out our budget request last year in April for this academic year. Since we don't have a document for end-of-semester tasks, I suggest we omit this line.

Even if it isn't documented, we always receive email heads-up from Club Office, so we can get around to documenting it later, if needed.

Reply:
Budget is done *only* at the end of spring semester

Reply:
Should we change it to verify budget @jwflory  @Tjzabel? I think that it would be important to have since we are doing it this semester.

Reply:
@jrtechs Nice idea – this is a good revision. :+1: +1 to revising to verify budget allocation.